wpeverest/user-registration
# UR-830 Feature -  Email preview

### All Submissions:






### Changes proposed in this Pull Request:



Previously, We do not have an email preview feature for individual email content notifications. This PR will add an email preview option. Also, this PR implements a new default template which will also be used for not only previewing emails but while sending emails from the plugin also.

### How to test the changes in this Pull Request:

1. Navigate to User registration settings then Email Tab.
2. Go to configure the email of each email type.
3. Make changes and save the content then check email preview is working properly or not.

### Types of changes:




### Other information:




### Changelog entry

> Feature -  Email preview.
